ludwig/features/base_feature.py

```diff
@@ -184,56 +181,52 @@ def create_preproc_module(metadata: Dict[str, Any]) -> torch.nn.Module:
 class OutputFeature(BaseFeature, LudwigModule, ABC):
     """Parent class for all output features."""
 
-    def __init__(self, feature: Dict[str, Any], other_output_features: Dict[str, "OutputFeature"], *args, **kwargs):
+    def __init__(
+        self,
+        feature: Union[BaseOutputFeatureConfig, dict],
+        other_output_features: Dict[str, "OutputFeature"],
+        *args,
+        **kwargs,
+    ):
         """Defines defaults, overwrites them based on the feature dictionary, and sets up dependencies.
 
         Any output feature can depend on one or more other output features. The `other_output_features` input dictionary
         should contain entries for any dependent output features, which is accomplished by constructing output features
         in topographically sorted order. Attributes of any dependent output features are used to properly initialize
         this feature's sizes.
         """
-        super().__init__(*args, feature=feature, **kwargs)
+        feature = self.load_config(feature)
```

Comment:
Just a quick question, but why do we load the config in the specific input/output features and the base output feature, but not in the base input feature?

Reply:
This is vestigial code - My first instinct was to `load_config` in the base class, but then I changed my mind.  `load_config` in `OutputFeature` doesn't do anything right now, will remove.